### PR TITLE
[NETBEANS-4908] Enable run/debug single when the suitable task is provided by the Gradle project.

### PR DESCRIPTION
**Pull request by lkishalmi**
Well it is a bit more than I thought at first. At the start I would have checked only the existence of runSingle task, though if I remember someone would have liked run task defined for certain executable class in his project. With this pr run.single action can be redefined to use a ```run${selectedClassName}``` task.

### CODE REVIEW

File: extide/gradle/test/unit/src/org/netbeans/modules/gradle/api/execute/RunUtilsTest.java [hunk 1]
```diff
@@ -70,4 +72,15 @@ public void testIncludeOpenProjects2() throws Exception {
         assertEquals(0, params.size());
     }
 
+    public void testEvaluateArgs1() throws Exception {
+        FileObject a = createGradleProject("projectA",
+                "apply plugin: 'java'\n", "");
+        Project prjA = ProjectManager.getDefault().findProject(a);
+        ProjectTrust.getDefault().trustProject(prjA);
+        openProject(a);
+        String[] args = RunUtils.evalueteActionArgs(prjA, null, "Project: ${projectPath}${projectName}", Lookups.singleton(prjA));
```
Comment (JaroslavTulach) [inline extide/gradle/test/unit/src/org/netbeans/modules/gradle/api/execute/RunUtilsTest.java:81]: Typo in `evalueteActionArgs`.

File: extide/gradle/src/org/netbeans/modules/gradle/spi/actions/ProjectActionMappingProvider.java [hunk 2]
```diff
@@ -17,13 +17,14 @@
  * under the License.
  */
 
-package org.netbeans.modules.gradle.actions;
+package org.netbeans.modules.gradle.spi.actions;
 
 import java.util.Set;
 import org.netbeans.modules.gradle.api.execute.ActionMapping;
 
 /**
```
Comment (JaroslavTulach) [inline extide/gradle/src/org/netbeans/modules/gradle/spi/actions/ProjectActionMappingProvider.java:25]: Consider adding better Javadoc. Description/example of "where/how to register this interface" was something I'd have expected here.

**Review by JaroslavTulach — APPROVED** [review]
Please fix the typo. Otherwise OK.

However I am not sure how can I benefit from this change in the `java.lsp.server` and/or the associated `nbcode` suite? Preferably the language server support is to be project type independent. Clearly `java.lsp.server` shall not depend on Gradle. But of course, I can add an integration module like `nbcode/gradleintegration` to the suite, if that helps to achieve our goals.

File: extide/gradle/test/unit/src/org/netbeans/modules/gradle/api/execute/RunUtilsTest.java (hunk 1 above)
Comment (lkishalmi) [reply extide/gradle/test/unit/src/org/netbeans/modules/gradle/api/execute/RunUtilsTest.java]: Thanks for spotting the typo!
Well, I do not know if this support ```java.lsp.server``` in any way. That would be your call to decide.

**Pull request merged**